### PR TITLE
17 devise styling

### PR DESCRIPTION
**Pull request by yndajas**
We only have a [design for the log in page](https://xd.adobe.com/view/85f38f01-ca5a-4c6b-a393-28b15a7e278b-d0a0/screen/c651bdf9-a319-488f-ad72-502e860a704a/specs), but I've also styled the registration, forgotten password, unlock and confirmation pages

# Testing

Make sure you're logged out, then:
1. visit [/criteria/new](http://localhost:3000/criteria/new), which should redirect to the sign in page with an error message
2. fill in the form with a bad email and password (i.e. not test@test.com / password or test2@test.com / password) and try to sign in to reveal another error message
3. go to [/users/sign_up](http://localhost:3000/users/sign_up)
4. go to [/users/password/new](http://localhost:3000/users/password/new)
5. go to [/users/confirmation/new](http://localhost:3000/users/confirmation/new)
6. go to [/users/unlock/new](http://localhost:3000/users/unlock/new)

### CODE REVIEW

File: app/assets/stylesheets/components/_devise.scss [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+    color: $grey-xxdark;
+    padding: rem-calc(185) rem-calc($content-gutter) 0 rem-calc($content-gutter);
+    max-width: rem-calc(575 + $content-gutter * 2);
+  }
+
+  &__title {
+    font-family: $header-font;
+    font-size: rem-calc(48);
+    margin-bottom: rem-calc(32);
+  }
```
Comment (RainbowGiantSquid) [inline app/assets/stylesheets/components/_devise.scss:84]: On the mocks margin here is 30, but it still looks very spced out. I reckon we can leave it is if you recon it's ok.
image

**Review by RainbowGiantSquid — APPROVED** [review]
nice! Just a couple of comments but I think we can merge it as is.

**Review by sergiomarrocoli — APPROVED** [review]
looks good to me. when the screen size is the same as the docs, the margins around the form fields looks right, on a larger screen it stretches out a bit. I think that's just a tweak we can worry about later though, if it's questioned

**Comment by RainbowGiantSquid** [comment]
Merging - can do nitt gritty stuff later on :)

**Pull request merged**